scripts/builld-sign-release-images.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
```

Comment:
typo in the file name: `builld-sign-release-images`

Reply:
thanks :)